bokeh/models/tools.py

```diff
@@ -513,6 +513,8 @@ class SaveTool(ActionTool):
 
     '''
 
+    filename = Nullable(String)
```

Comment:
This needs `help=...`, explaining whether an extension is needed and what happens when `None` value is used.

Reply:
Added in 28c8ac3ec38e7b8473934fe3166514ff6a3070d1 but feel free to propose changes.